lib/ruby2d/window.rb

```diff
@@ -236,6 +223,36 @@ def add_object(o)
         false
       end
     end
-    
+
+    def events_store(event_type)
```

Comment:
Although, now that I thought of it more, maybe it's better to stick to case. It's more verbose, it's very readabla, and the performance always comes in handy.